source/install/desktop-managed-resources.rst

```diff
@@ -0,0 +1,40 @@
+
+Desktop - Managed Resources
+===========================
+
+The Mattermost Desktop App supports managed resources. A managed resource can be any service available on the same hostname using the same protocol as the Mattermost server.
+
+.. Note::
+    Using this feature requires a `custom build of the Mattermost Desktop App <https://docs.mattermost.com/deployment/desktop-app-deployment.html>`_.
+
+When adding the path of a managed resource to `src/common/config/buildConfig.js <https://github.com/mattermost/desktop/blob/master/src/common/config/buildConfig.js>`_ it opens as a pop-up window in the Mattermost Desktop App. 
+
+In the below example we add the managed resource ``/video``.
```

Comment:
Suggestion:
As per the above comment, this would work really well if it was part of a larger example of where one would find this section as it would add some context and perhaps some clarity around the goal and outcome.

Reply:
"This" refers to the code example below.

Reply:
@Willyfrog, would you be open to reviewing the code below and confirming that it is accurate? If you would be able to confirm the path the user would follow to locate the area where they're adding the content that would also be great. :)

Reply:
Yes,  it is accurate 😊